src/reducers/profile-view.js

```diff
@@ -721,7 +729,12 @@ export const selectorsForThread = (
       getCallNodeInfo,
       UrlState.getImplementationFilter,
       UrlState.getInvertCallstack,
-      CallTree.getCallTree
+      (...args) => {
+        const callTree = CallTree.getCallTree(...args);
+        // Allow the user access the last viewed call tree via the web console.
+        (window: any).callTree = callTree;
+        return callTree;
+      }
     );
 
     // The selectors below diverge from the thread filtering that's done above;
```

Comment:
Here is an alternate suggestion to do this, tell me what you think: use a connected Component instead of coupling this inside the reducers.

```javascript
type StateProps = {|
  +profile: Profile,
  +filteredThread: Thread,
  +callTree: CallTree,
|};

class ExposingComponent extends PureComponent<StateProps> {
  render() {
    return null; // maybe it's the default if we don't even implement render ?
  }

  componentDidUpdate() {
    // instead of unconditionally assigning all values you may want to compare the values.
    // But I don't think this is necessary.
    Object.assign(window, this.props);
  }
}

const connectOptions: ExplicitConnectOptions<{||}, StateProps, {||}> = {
  component: ExposingComponent,
  mapStateToProps: state => ({
    profile: getProfile(state),
    filteredThread: selectedThreadSelector.getFilteredThread(state),
    callTree: selectedThreadSelector.getCallTree(state)
  }),
};

export default explicitConnect(connectOptions);
```

(I haven't looked up the actual names so I may have typos :) )

The main advantage of this is that it's very very easy to know what we expose, this is very clear, everything at one place.